### PR TITLE
MX : Add Miami TV Latino

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Mexican channel according to their Facebook once they talk about this "Latino" channel.
However, the original Miami TV is based apparently in Miami, Florida, USA.

Their website (https://miamitvhd.com) do not seem to say much about their location.
https://facebook.com/watch/miamitv/

### CODE REVIEW

File: channels/mx.m3u [hunk 1]
```diff
@@ -65,3 +65,5 @@ https://5ca3e84a76d30.streamlock.net/gpacifico4/smil:mazatlan.smil/chunklist_w74
 https://5ca3e84a76d30.streamlock.net/gpacifico2/mochis.smil/chunklist_w52505888_b1428000_sleng.m3u8
 #EXTINF:-1 tvg-id="TVPobregon.mx" tvg-name="TVP obregon" tvg-country="MX" tvg-language="" tvg-logo="https://i.imgur.com/mw4992J.png" group-title="",TVP obregon
 https://5ca3e84a76d30.streamlock.net/gpacifico3/obregon.smil/chunklist_w888631267_b1428000_sleng.m3u8
+#EXTINF:-1 tvg-id="MiamiTVLatino.mx" tvg-name="Miami TV Latino" tvg-country="MX" tvg-language="" tvg-logo="https://statics.yowi.tv/files/c74681cb7f75db303b818a931702600c44a9692d" group-title="XXX",Miami TV Latino
+https://59ec5453559f0.streamlock.net/canal18/smil:canal18/playlist.m3u8
```
Comment (freearhey) [inline channels/mx.m3u:69]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/mx.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/mx.m3u:69]: I'm looking at it.

**Comment by freearhey** [comment]
All other Miami TV links are located in US playlist:
https://github.com/iptv-org/iptv/blob/705b0c3b132e0b9f9ec8d7a09cf8cb9562bc4a31/channels/us.m3u#L1197-L1216

so I think it's better to put this link there, too.

**Comment by LaneSh4d0w** [comment]
You're right, let me move that.

**Comment by LaneSh4d0w** [comment]
See #2830

**Pull request closed without merging**